README.md

```diff
@@ -2,6 +2,7 @@
 
 Rocketchat for yunohost
 
+[![Integration level](https://dash.yunohost.org/integration/rocketchat.svg)](https://ci-apps.yunohost.org/jenkins/job/rocketchat%20%28Community%29/lastBuild/consoleFull)  
```

Comment:
can you please change `https://ci-apps.yunohost.org/jenkins/job/rocketchat%20%28Community%29/lastBuild/consoleFull` to `https://ci-apps.yunohost.org/ci` then? Because all other links are not available without authentication.